index.html

```diff
@@ -16,7 +16,13 @@
     <header>
         <div class="logo"><a id="logo" href="#logo">My Logo</a></div>
         <nav>
-            <button type="button" class="hamburger" id="openBtn"><img src="img/hamburger.png" alt="Menu icon"></button> 
+            <button type="button" class="hamburger" id="openBtn"><img src="img/hamburger.png" alt="Menu icon"></button>
+            <ul class="links">
+              <div class="close-button"><button id="close"><img src="img/close.png" alt="close button"/></button></div>
+              <li><a href="#Portfolio" id="linkNav-port">Portfolio</a></li>
+              <li><a href="#About" id="linkNav-about">About</a></li>
+              <li><a href="#Contact" id="linkNav-cont">Contact</a></li>
+            </ul> 
```

Comment:
- Hi @jrraul4 :wave:
- [ ] Your styling is really great but you do have a minor issue with managing the responsiveness of your `Works section` and the `Contact section`. Kindly check the [video](https://watch.screencastify.com/v/BBq05WDvFhzYGOCYhyZZ) attached and compare it with the Figma design to ensure your aesthetics looks impressively usable to all kinds of users on all types of devices
- [ ] Another I spoke about is a broken image -Github icon on your landing page: 
![image](https://github.com/jrraul4/Portafolio-Personal/assets/99715640/6c000589-10e9-46d1-b776-dfcc95f53f38)